src/app/Common/Services/Domain/BaseDocumentService.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Threading.Tasks;
+using Common.Dal;
+using Common.Dal.Documents;
+using Common.Dal.Interfaces;
+using Common.Services.Domain.Interfaces;
+
+namespace Common.Services.Domain
+{
```

Comment:
Just curious why in this place you didn't use new bracket-less namespace as you proposed in previous PR?